lib/KBreadcrumbs.vue

```diff
@@ -323,77 +115,39 @@
     width: 100%;
     max-width: $crumb-max-width;
     overflow: hidden;
+    font-weight: bold;
+    text-decoration: none;
     text-overflow: ellipsis;
     white-space: nowrap;
     vertical-align: bottom;
   }
-
-  .breadcrumbs-dropdown-wrapper {
-    display: inline-block;
-
-    &::after {
-      margin-right: 8px;
-      margin-left: 8px;
-      content: '›';
-    }
-  }
-
-  .breadcrumbs-dropdown {
-    @extend %dropshadow-2dp;
-
-    position: absolute;
-    z-index: 8;
-    max-width: 100%;
-    padding: 16px;
-    font-weight: bold;
+  .breadcrumbs-divider {
+    margin-right: 8px;
+    margin-left: 8px;
+    color: #000000;
```

Comment:
Lets avoid burned color, and lets use the KDS color tokens instead https://design-system.learningequality.org/colors/#tokens-text